routes/web/project/vm.rb

```diff
@@ -27,6 +27,15 @@ class CloverWeb
         enable_ip4: r.params.key?("enable-ip4")
       )
 
+      UsageRecord.create_with_id(
```

Comment:
same thing, should we move this to nexus?

Reply:
I agree with that. Because we can create new vm via UI, API or pry.

Reply:
Yep moved it to the nexus.